src/Parallelotope.jl

```diff
@@ -0,0 +1,125 @@
+export Parallelotope
+
+"""
+    Parallelotope{N<:Real, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}}
+     <: AbstractZonotope{N}
+
+Type that represents a parallelotope.
+
+### Fields
+
+- `offset`     -- vector; each element is the offset of the corresponding constraint
+- `directions` -- matrix ``D``; each row is the direction of two corresponding
+                  constraints
+
+### Notes
+``D_{i} = -D_{i+n}`` for ``i = 1, ..., n`` with ``n`` rows in ``D``
```

Comment:
```suggestion

``D_{i} = -D_{i+n}`` for ``i = 1, ..., n`` with ``n`` rows in ``D``
```

Reply:
Can you add some explanation or a reference? I do not understand how this set representation works.